samples/regexp.bolt

```diff
@@ -0,0 +1,47 @@
+path /  {
+  read() = true;
+  write() = true;
+}
+
+path /ss is SocialSecurity;
+path /integer is IntegerString;
+path /float is FloatString;
+path /int is Integer;
+path /alpha is Alpha;
+path /year is Year;
+path /date is ISODate;
+
+type SocialSecurity extends String {
+  validate() = this.test(/^\d\d\d-\d\d-\d\d\d\d$/);
+}
+
+// Only an integer string
+type IntegerString extends String {
+  validate() = this.test(/^\d+$/);
+}
+
+// A floating point number (or integer).
+type FloatString extends String {
+  validate() = this.test(/^\d+(\.\d*)$/);
```

Comment:
Not sure how much you care (since this is a sample), but:
1. Did you maybe intend to add a ? after the (\.\d*) clause so that things like "0" and "101" count as floats?
2. This won't work for negative floats or floats represented in exponential notation. ;-)  If you want it to be complete, I'm guessing there are decent regexes floating around on the internet already. :-)